frontend/apps/reader/modules/readertoc.lua

```diff
@@ -644,6 +644,10 @@ function ReaderToc:expandParentNode(index)
 end
 
 function ReaderToc:onShowToc()
+    if self.view.inverse_reading_order then
```

Comment:
This would have to be confirmed by manga readers: with japanese books with RTL page turning, do users really expect to have TOC mirrored ?
Should it be another setting, individual or additional to inverse_reading_order ?

Reply:
see next comment